### ISSUE TITLE
can't get it to work

### ISSUE THREAD

**Issue by tanloverluke** [monkeysuffrage/advanced_html_dom#3]
Currently using simple html dom and I was under the impression that only 2 changes were necessary:
1) replace simple_html_dom.php contents with those from advanced_html_dom.php
2) change `$html = new simple_html_dom();` to `$html = new AdvancedHtmlDom()`
However this isn't working . It would be very useful if you can include examples where your code works, because at the moment it does not. If you included example code it would make it so much easier to debug.
For your information I am using curl to grab the page contents.

**Comment by monkeysuffrage** [comment]
Try using str_get_html or file_get_html. I don't consider this an issue.

**Comment by nekromoff** [comment]
I agree that the README is pretty lousy. Definitely not friendly drop-in replacement.

**Comment by monkeysuffrage** [comment]
@Galimatias - feel free to contribute.

**Comment by nekromoff** [comment]
No, thank you, not contributing to basically an undocumented library.

**Comment by monkeysuffrage** [comment]
Read the simple html dom documentation in that case. All the common simple
html dom methods are available in this library.

**Comment by LuxCave** [comment]
Hello, I can help in contributing and document the library.
